corehq/apps/reports/commtrack/data_sources.py

```diff
@@ -434,6 +445,8 @@ def get_data(self):
 
             for spoint_id, loc_id in spoint_loc_map.items():
                 loc = locations[loc_id]
+                if loc.is_archived:
+                    continue
```

Comment:
I'm assuming path returns a path from root to the parent of the location. could you use: http://django-mptt.github.io/django-mptt/models.html#get-ancestors-ascending-false-include-self-false